src/pymordemos/era.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+import numpy as np
+import matplotlib.pyplot as plt
+from typer import run
+
+from pymor.core.logger import set_log_levels
+from pymor.models.iosys import LTIModel
+from pymor.reductors.bt import BTReductor
+from pymor.reductors.era import ERAReductor
+from pymor.tools.random import get_rng, new_rng
+
+
+def example_system(order, sampling_time=1):
+    """Construct an example system.
+
+    Parameters
+    ----------
+    order
+        The order of the constructed LTI system.
+    sampling_time
+        The sampling time of the LTI system.
+
+    Returns
+    -------
+    sys
+        The constructed |LTIModel|.
+    """
+    rng = get_rng()
+    A = np.eye(order) * 1e-3
+    B = rng.random((order, 1)) / 2
+    C = rng.random((1, order)) / 2
+
+    A[:2, :2] = np.array([[0.8876, 0.4494], [-0.4494, 0.7978]])
+    A[2:4, 2:4] = np.array([[-0.6129, 0.0645], [-6.4516, -0.7419]])
+    B[:4, :] = np.array([[0.2247], [0.8989], [0.0323], [0.1290]])
+    C[:, :4] = np.array([0.4719, 0.1124, 9.6774, 1.6129])
+    D = np.array([0.9626])
```

Comment:
Where do these numbers come from? (It might be good to document it in a comment.)

Do these numbers have to be used, or is there an example with nicer numbers?

Reply:
The numbers are taken from https://ieeexplore.ieee.org/document/508900 (Example III C). I was trying to create a system that is of oder 4, approximatively. When trying with simpler numbers, the transfer function was less interesting.

I didn't think a bib entry was warranted, since they are just some numbers. Should I add a comment?

Reply:
Ok, sounds reasonable. I also don't think it should be added as a bib entry, but a comment with the link would be good for future reference.